db/seeds.rb

```diff
@@ -5,3 +5,156 @@
+                                   year_id: year.id,
+                                   data_type_id: f10.id
+    end
+  end
+end
+
+def test_c
+  f1 = DataType.create! name: 'f1'
+  f2 = DataType.create! name: 'f2'
+  f3 = DataType.create! name: 'f3'
+  f4 = DataType.create! name: 'f4'
+  f5 = DataType.create! name: 'f5', formula: 'f4 * f3'
+  f6 = DataType.create! name: 'f6', formula: 'f4 + 10'
+  f7 = DataType.create! name: 'f7', formula: 'if (f1 < 5, 1, 0)'
+  f8 = DataType.create! name: 'f8', formula: 'f4 + 5'
+  f9 = DataType.create! name: 'f9', formula: 'f1 + f2 + f3 + f4 + f5 + f6 + f7 + f8 + f10'
+  f10 = DataType.create! name: 'f10', formula: 'f4 * f3'
+  3.times do
+    project = Project.create! name: FFaker::Name.name,
+                              acres: Random.new.rand(1..69),
+                              date_closed: FFaker::Time.date,
+                              restricted_endowment: Random.new.rand(1..69),
+                              cap_rate: Random.new.rand(1..69),
+                              admin_rate: Random.new.rand(1..69),
+                              total_upfront: Random.new.rand(1..69),
+                              years_upfront: Random.new.rand(1..69),
+                              earnings_begin: FFaker::Time.date
+    for yyyy in 2068..2070
+      year = Year.create! date: yyyy, project_id: project.id
+      f1_value = DataValue.create! value: 5.0,
+                                   year_id: year.id,
+                                   data_type_id: f1.id
+      f2_value = DataValue.create! value: 10.0,
+                                   year_id: year.id,
+                                   data_type_id: f2.id
+      f3_value = DataValue.create! value: 15.0,
+                                   year_id: year.id,
+                                   data_type_id: f3.id
+      f4_value = DataValue.create! value: 20.0,
+                                   year_id: year.id,
+                                   data_type_id: f4.id
+      f5_value = DataValue.create! value: 0.0,
+                                   year_id: year.id,
+                                   data_type_id: f5.id
+      f6_value = DataValue.create! value: 0.0,
+                                   year_id: year.id,
+                                   data_type_id: f6.id
+      f7_value = DataValue.create! value: 0.0,
+                                   year_id: year.id,
+                                   data_type_id: f7.id
+      f8_value = DataValue.create! value: 0.0,
+                                   year_id: year.id,
+                                   data_type_id: f8.id
+      f9_value = DataValue.create! value: 0.0,
+                                   year_id: year.id,
+                                   data_type_id: f9.id
+      f10_value = DataValue.create! value: 0.0,
+                                   year_id: year.id,
+                                   data_type_id: f10.id
+    end
+  end
+end
+
+test_c
```

Comment:
No prob. Again, sticking a `TODO` or a disclaimer here is a good idea since I see PRs as code that will eventually make it into the app where someone else will have to look at it someday.